Resources/Prototypes/DeltaV/Access/justice.yml

```
@@ -0,0 +1,19 @@
+- type: accessLevel
+  id: ChiefJustice
+  name: id-card-access-level-cj
+
+- type: accessLevel
+  id: Prosecutor
+  name: id-card-access-level-prosecutor
+
+- type: accessLevel
+  id: Justice
+  name: id-card-access-level-justice
+
+- type: accessGroup
+  id: Justice
+  tags:
+  - Justice
+  - Prosecutor
+  - ChiefJustice
+  - Lawyer
```

```suggestion
  - Lawyer
  
```
line at the end readability standards